### DISCUSSION TITLE
I do not want "For You" to be the default

### DISCUSSION THREAD

**Discussion by PetrGlad** [community/community#66133] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

It seems that GitHub strives to mimic a dopamine cranking social network, but to me it is actually about the code. The key difference is that notifications in GitHub require a lot more attention, say, serious bug or security problem in one of the projects I am interested in could easily take hours to be taken care of, and diluting it with a nice-to-have stream of random items would drain too much energy. I could expect something like this elsewhere, but there I know I could just skim hundreds of posts without really paying attention.
There is no problem for me in some sort of "Explore" function that is somehow match my interests but that should be on demand, not the default.

**Comment by tekknolagi** [comment]
I also just have lost a lot more information. I like reading about people's commits and PRs and stuff. That seems to be all gone.

**Reply by FoxxMD** [reply]
This is my biggest concern. None of the activities are from people *I actual follow* anymore.

**Reply by Siedlerchr** [reply]
Yes, this was my primary interest as well. I only care for the things people I follow do

**Reply by jeremyckahn** [reply]
+1. This feels completely broken. This is a significant regression in functionality. :(

**Reply by mryechkin** [reply]
Yup.. it's nearly useless to me now. Absolutely hate this tone-deaf change.

**Reply by derrod** [reply]
+1, I watched repos to see activity (commits, issues, PRs) in my feed, now it's gone. All I get is some month-old event that isn't even relevant based on the filter description.

I don't know why GitHub did this again after the abject failure that was their previous attempt at making this the only feed, and the unequivocally negative feedback they've gotten every time.

**Reply by medewitt** [reply]
And it virtually unreadable using a phone (especially when I'm interested in commits and bugs for particular apps)

**Reply by bytefish** [reply]
You can find the old dashboard at: https://github.com/dashboard-feed.

**Comment by yannikbloscheck** [comment]
I can't believe how after all the "For you" feed criticism the decision was made to basically make it the only feed, but in even more distracting

**Reply by po5** [reply]
Glad they take feedback! haha.......

**Comment by eatonphil** [comment]
Don't mind if it's an option. But the chronological view should be an option too. And I'd like to be able to have chronological as the default.

**Comment by Zeioth** [comment]
I'm gonna be very clear. If Github keep pushing this shit onto us, I go back to bitbucket, or gitlab, or any other.

read my lips: 

* I NEED - THE TOOL - FOR WORK.
* DON'T MAKE MY WORK → FUCKING → SOCIAL MEDIA.
* LET ME WORK → IN FUCKING PEACE.

I hope that was clear. Thank you.

**Comment by klarkc** [comment]
I agree, If I need a social network, X is already there. Why transform GitHub in yet another social network? Please bring my own subscribed content, not random nor "someone else" filtered content.

**Comment by triska** [comment]
I am using Github for working with colleagues on several different projects, and I would appreciate a clutter-free interface that immediately shows the changes to projects I follow, and most recent commits and actions of my colleagues.

Could you therefore please restore the previous version of the feed which exhibited these characteristics? Thank you a lot!

**Comment by bastimeyer** [comment]
Just because I have starred some cool projects in the past doesn't mean that I'm interested in seeing them in my news feed. My news feed is supposed to be about the projects I really care about and which I'm involved in, so I don't miss important stuff, in addition to my notifications.

I really dislike this "For You" nonsense which is just random noise. It doesn't have any value for me.

From testing the "Show less activity like this" option on each unwanted "news" item, it doesn't appear to work, so what this "For You" "newsfeed" actually does is make me think about unstarring everything and unfollowing everyone.

It's just bad and doesn't belong on GitHub, at least not as the default option.

**Comment by mpaperno** [comment]
The personal front page turned from fairly concise information about stuff I actually work on/with to some ridiculous collection of "news" about what _other_ (mostly random!) people do.  How is that possibly the same thing?  Or actually useful?

Bring the old view back, please.  There's no way to replicate the same view with the limited "filters" available.

**Comment by cj-mills** [comment]
The new home feed is worse than nothing at all. I tried using the filters to approximate the original setup, but the information layout is still worse. I'm technically getting less information but feeling overloaded because of the presentation.

**Comment by leeoniya** [comment]
no one asked for this.

literally. no one.

i wonder if the PMs who decide to implement these absurd changes actually use the thing they're destroying for actual work. all signs clearly point to "no".

the feed went from being good, to 50% useless, to now 100% useless in about 3 years. truly remarkable. this is what happens when the software is no longer guided by engineers for fellow engineers, but by managers for their managers.

**Comment by xt0rted** [comment]
I'll give the same feedback I gave last time you tried to force me to use this as my dashboard, I don't care about 99% of what's in this feed. What I care about is seeing who forked or starred my repos, who sponsored me, new issues opened in my repos, and basic info about people I follow like their forks or new repos.

I want the original feed that made sense for how I, and others, use this site. You removed the term `social coding` from your feature page at the end of 2019, so remove it from my dashboard too and let me see MY activity.

Given your push for GHE users I fail to see how this dashboard would appeal to any of those customers/users either. Shouldn't the dashboard be something that helps you, not something that's a huge distraction?

**Comment by jevinskie** [comment]
Stop dicking around with the activity feed. You screwed it up the last two times you tried and have once again screwed it up for a third time without any option of opting out (again). You can’t fix what isn’t broken!

**Comment by jeremy-coleman** [comment]
Cant wait for the linkedin integration

**Comment by rabanti-github** [comment]
I don't care about "for you" at all. These recommendations may be repos in completely unrelated programming languages, a topic that intersects one of my repos "accidentally", or hot topics that basically are deemed to be hyped for everyone.
No, sorry, this is not YouTube where we consume content. This is code. And at least for me, I am not interested at all in repos that are not actually used by me.
And by the way... It looks like "User XYZ starred your repo ABC" was removed too. If you want be the YouTube of software, bring back this to give us our daily dopamine boost... Yeah, sounds quite narcistic, but it is lik it is :-D

**Comment by kiwifb** [comment]
All the stuff I wanted in the feed is gone and I cannot find a setting to at least re-include it (and I tried). The landing page of github is now useless to me. Sure coding can be a social activity, but it is not the same social settings as facebook or twitter nor should it be.

**Comment by kurtmckee** [comment]
I'm disappointed to be confronted with a blank default page.

I had to disable all of the categories for lack of valuable information, but I'm left with a blank default page.

Please revert this change.

**Reply by bytefish** [reply]
I can’t even see, when someone creates an issue in one of my repositories. It’s quite hilarious actually, that something like this is being rolled out. 

I guess it’s time for performance evaluations at GitHub, and someone had to get a feature out. 🤷‍♂️

**Comment by utybo** [comment]
I'm a small developer on GitHub and used my "Following" feed for two things:

- See when people follow me, star my stuff, etc. to see if someone likes what I'm doing.
- See what people I follow star or create.

It seems the new tab does include this information, but completely out of order. What I actually use the homepage for is akin to some "notification feed", not a "social-like" feed. Please at least provide an option to sort the provided filters in chronological order.

**Comment by kiwifb** [comment]
I have now decided to remove all the notifications offered to me in the new menu selection in protest. Now I get a blank page but if someone is monitoring those settings, it should send a message if enough of us do it.

**Reply by leeoniya** [reply]
trust me, no one is monitoring anything. there has been a huge backlash here in discussions with every rollout of social features to the feed for the past three years. they are 100% tone-deaf.

**Reply by kiwifb** [reply]
Sad, I still think just disabling the feed is the thing to do.

**Comment by bytefish** [comment]
Does anyone know of a thread, where the GitHub Product Manager or Product Owner explains what exactly has been broken in the previous version and why this update was forced upon without running a beta? I am really curious, because my feed is now next to useless. 

I’ve always thought a “filter” filters for something. And removing all filters on a result set should return an unfiltered result set. So this is what my feed looks like, when I remove the filters. 

I maintain various projects by the way and the information density is just amazing:

![image](https://github.com/community/community/assets/239373/42a9b76b-fcb0-4891-b3c3-708a3dd5ba26)

Does the 8 filters mean, that I have 8 filters activated or 8 filters are available? Top-notch usability.

Edit: The “official” announcement is over at https://github.com/orgs/community/discussions/65343.

**Comment by Brendonovich** [comment]
In principle I don't have any objections to having a more social-style For You feed, I even like looking at it on mobile sometimes and have found some cool projects through it.
Making it the only option, though? Not a fan.
Ever since For You was introduced I came to the conclusion that 99% of the time I only want to have my homepage feed to contain information about me. Who is starring and forking my repos, following and sponsoring me, those are the thing that actually add value to me _daily_ as an OSS maintainer.
To just remove that entirely is disheartening. If I wanted to see information about other users and repos, For You was a great way to do it! But when it was added I knew there was a chance it might become the only option, and lo and behold here we are now.
I'll be keeping all my feed filters unticked since none of them are valuable to me very often, and hoping for a reintroduction of the old me-centric feed, though I'm not holding out hope.

**Comment by HHHHHHHHHHHHHHHHHHHHHCS** [comment]
I may pay more attention to who star/forks me and my projects.
I care about which projects the people I follow star/create,
rather than "Recommended for you".

**Comment by wangrongding** [comment]
# Github Old Feed  

💪 Using this script you can restore the old version of the feed.
https://github.com/wangrongding/github-old-feed  

<table>
  <tr>
      <td width="50%" align="center"><b>Before</b></td>
      <td width="50%" align="center"><b>After use</b></td>
  </tr>
  <tr>
     <td><img src="https://assets.fedtop.com/picbed/1694066442147.jpg"/></td>
     <td><img src="https://assets.fedtop.com/picbed/1694066437621.jpg"/></td>
  </tr>
</table>

**Comment by bytefish** [comment]
If anyone wants to access the old dashboard, you can still find it at: https://github.com/dashboard-feed. This gives me useful information at least. Does anyone know what’s going to happen with the RSS feed for the dashboard?

**Reply by kiwifb** [reply]
This is an exceptionally helpful post! Thank you, thank you!

**Comment by mipmip** [comment]
Even for a marketeer this is a bad decision.

**Comment by ettaboyle** [comment]
_We've added the following update to the [Updates to your GitHub Feed](https://github.com/orgs/community/discussions/65343#top) Discussion Announcement originally posted August 29, 2003._


## _Sept. 12th Update_
_With the latest feed update on September 6, 2023, we made changes to the underlying technology of the feed to improve overall platform performance. As a result, we removed the functionality for “push events for repositories a user is subscribed to”. We don’t take these changes lightly, but as our community continues to grow tremendously, we have to prioritize our availability, user experience, and performance._ 


_Thanks to feedback from the community, we have updated our [changelog](https://github.blog/changelog/2023-09-06-updates-to-your-github-feed/) to clarify and have fixed the following bugs from the initial September 6th release:_


- _We were showing releases for organizations that you follow, instead of only showing them for repositories that you watch or star. This is now fixed._ 
- _We have also addressed a problem where 'Followed You' cards were sometimes appearing out of chronological order._
- _We do not plan to re-include “push events for repositories a user is subscribed to” for the reasons listed above, but we will continue to address feedback as it aligns with our platform goals._


_We understand that many of you are upset with the recent changes to your feed. We should have done a better job communicating recent changes and how those decisions relate to our broader platform goals. Your continued feedback is invaluable as we evolve and continue to strive to provide a first-class developer experience that helps every developer be happier and more productive._

**Comment by ettaboyle** [comment]
_We've added the following update to the [Updates to your GitHub Feed](https://github.com/orgs/community/discussions/65343#top) Discussion Announcement originally posted August 29, 2003._


## _Sept. 12th Update_
_With the latest feed update on September 6, 2023, we made changes to the underlying technology of the feed to improve overall platform performance. As a result, we removed the functionality for “push events for repositories a user is subscribed to”. We don’t take these changes lightly, but as our community continues to grow tremendously, we have to prioritize our availability, user experience, and performance._ 


_Thanks to feedback from the community, we have updated our [changelog](https://github.blog/changelog/2023-09-06-updates-to-your-github-feed/) to clarify and have fixed the following bugs from the initial September 6th release:_


- _We were showing releases for organizations that you follow, instead of only showing them for repositories that you watch or star. This is now fixed._ 
- _We have also addressed a problem where 'Followed You' cards were sometimes appearing out of chronological order._
- _We do not plan to re-include “push events for repositories a user is subscribed to” for the reasons listed above, but we will continue to address feedback as it aligns with our platform goals._

_We understand that many of you are upset with the recent changes to your feed. We should have done a better job communicating recent changes and how those decisions relate to our broader platform goals. Your continued feedback is invaluable as we evolve and continue to strive to provide a first-class developer experience that helps every developer be happier and more productive._

**Reply by kiwifb** [reply]
While I appreciate the bug fixes, the exclusion of "push events for repositories a user is subscribed to" in the feed is not satisfactory. To put it in business term.
From what we can see from the feedback, if you had done a proper consultation with stakeholder (i.e. the developers using github), you would have found it to be a business requirement. In short your delivery does not meet the business requirement of your users. I hope your paying users get better service.
